kokoro/release/python/windows/build_python_env.bat

```diff
@@ -0,0 +1,10 @@
+REM Prepend newly installed Python to the PATH of this build (this cannot be
```

Comment:
Why does this have to be in a separate script? Can you embed it in another existing script?

Reply:
done